src/hyperfine/benchmark.rs

```diff
@@ -278,8 +292,8 @@ pub fn run_benchmark(
     all_succeeded = all_succeeded && success;
 
     // Re-configure the progress bar
-    progress_bar.set_length(count);
-    progress_bar.inc(1);
+    if let Some(bar) = progress_bar.clone() { bar.set_length(count); }
+    if let Some(bar) = progress_bar.clone() { bar.inc(1); }
```

Comment:
same here. avoid the `.clone()`

Also. why not use a single `if`?

Reply:
updated

hmm, yeah, good point about single `if`